fe/fe-core/src/main/java/org/apache/doris/common/util/TimeUtils.java

```diff
@@ -256,7 +256,7 @@ public static String checkTimeZoneValidAndStandardize(String value) throws DdlEx
             Matcher matcher = TIMEZONE_OFFSET_FORMAT_REG.matcher(value);
             // it supports offset and region timezone type, "CST" use here is compatibility purposes.
             boolean match = matcher.matches();
-            if (!value.contains("/") && !value.equals("CST") && !match) {
+            if (!value.contains("/") && !value.equals("CST") && !value.equals("UTC") && !match) {
```

Comment:
With more and more time zone added, I think it is better to refactor this judgment to make it easy to maintain.

Reply:
The root case of the problem such as `PRC` Timezone is that TimeUtils check is only check by regex we given, maybe we should also check the timezone exists in `ZoneId.getAvailableZoneIds`?

Reply:
> The root case of the problem such as `PRC` Timezone is that TimeUtils check is only check by regex we given, maybe we should also check the timezone exists in `ZoneId.getAvailableZoneIds`?

This timezone feature is introduced by @HangyuanLiu , as he explains, the time zone represented by this time zone abbreviation may be uncertain. For example, CST can be regarded as multiple time zones, so there is no support for such abbreviations, only a few cases are supported.

Reply:
Yes.  the time zone represented by this time zone abbreviation may be uncertain.
Another reason is the timezone string may transfer to be, the library of timezone in be may not support short time zone.
So may be you should test UTC can run correctly in be time function ,such as from_unixtime